nginx/Dockerfile

```diff
@@ -144,19 +144,27 @@ ENV ACCESSLOG=/var/log/nginx/access.log \
     REAL_IP_HEADER="X-REAL-IP" \
     REAL_IP_PROXY_HEADER="X-REAL-IP" \
     REAL_IP_RECURSIVE="on" \
-    PROXY_TIMEOUT=60s \
-    PROXY_SSL_CERT=/etc/nginx/conf/server.crt \
-    PROXY_SSL_CERT_KEY=/etc/nginx/conf/server.key \
-    PROXY_SSL_DH_BITS=2048 \
-    PROXY_SSL_PROTOCOLS="TLSv1.2 TLSv1.3" \
+    PROXY_SSL=off \
+    PROXY_SSL_CERT=/etc/nginx/conf/proxy.crt \
+    PROXY_SSL_CERT_KEY=/etc/nginx/conf/proxy.key \
     PROXY_SSL_CIPHERS="ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384" \
-    PROXY_SSL_PREFER_CIPHERS=off \
+    PROXY_SSL_PROTOCOLS="TLSv1.2 TLSv1.3" \
     PROXY_SSL_VERIFY=off \
-    PROXY_SSL_OCSP_STAPLING=off \
+    PROXY_SSL_VERIFY_DEPTH=1 \
+    PROXY_TIMEOUT=60s \
     SERVER_NAME=localhost \
     SERVER_TOKENS=off \
+    SSL_CERT=/etc/nginx/conf/server.crt \
+    SSL_CERT_KEY=/etc/nginx/conf/server.key \
+    SSL_CIPHERS="ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384" \
+    SSL_DH_BITS=2048 \
+    SSL_OCSP_STAPLING=on \
     SSL_PORT=443 \
-    TIMEOUT=60s \
+    SSL_PREFER_CIPHERS=off \
+    SSL_PROTOCOLS="TLSv1.2 TLSv1.3" \
+    SSL_VERIFY=off \
+    SSL_VERIFY_DEPTH=1 \
+    KEEPALIVE_TIMEOUT=60s \
     WORKER_CONNECTIONS=1024 \
```

Comment:
I saw that you partially ordered alphabetically variable names. Why not just apply it everywhere for consistency?